### ISSUE TITLE
[Bug] `portfolio/po/load` Error: [Errno 2] No such file or directory: 'OpenBB_Parameters_Template_v1.0.0.xlsx'

### ISSUE THREAD

**Issue by colin99d** [OpenBB-finance/OpenBBTerminal#2918]
<img width="646" alt="Screen Shot 2022-10-19 at 2 05 17 PM" src="https://user-images.githubusercontent.com/72827203/196685861-da7aff6f-3c18-4b6a-b749-98c3860d4b66.png">

Note. This happened on the newest pip package build.

**Comment by montezdesousa** [comment]
Is `OpenBB_Parameters_Template_v1.0.0.xlsx` being prompted on your `load`?

![Screenshot 2022-10-19 at 16 18 45](https://user-images.githubusercontent.com/79287829/196733602-3337a6b0-2bfe-4a4b-a306-b3f80d34d309.png)

**Comment by simmonsj330** [comment]
@montezdesousa I believe that it is not finding the file, which is located in `openbb_terminal/miscellaneous/portfolio_examples/optimization`. I noticed this when running integration tests on a new build here: (https://github.com/OpenBB-finance/OpenBBTerminal/actions/runs/3286345893/jobs/5414350894). 

<img width="1210" alt="Screen Shot 2022-10-20 at 12 15 12 AM" src="https://user-images.githubusercontent.com/53658028/196854964-b036898f-1413-4985-82dd-9d3d8edd3571.png">

The integration tests is as followed and breaks when loading the file in the `params` menu in the 5th line.

```
portfolio
load --file Public_Equity_Orderbook.xlsx
po
params
file --file OpenBB_Parameters_Template_v1.0.0.xlsx
```

Also, when replicating it on the conda terminal, the file is not prompted:
<img width="570" alt="Screen Shot 2022-10-20 at 12 19 04 AM" src="https://user-images.githubusercontent.com/53658028/196856005-415044b1-503d-4413-980f-ee1e969c893e.png">

**Comment by hjoaquim** [comment]
This was solved by #2972 

![image](https://user-images.githubusercontent.com/48914296/197754290-98ad5382-a6c6-421c-8f55-fb65a3d367b1.png)
